README.md

```diff
@@ -2,10 +2,246 @@
+service "hzservice" deleted
+```
+
+You can also delete the Persistent Storage Claim by:
+
+```
+$ oc delete pvc hz-vc
+```
+
+If you don't do it, then the next time you run your application, the same storage will be re-used.
+
+# Custom Configuration
+
+In order to use a custom Hazelcast configuration (or custom domain JARs), you need to copy them into the Persistent Volume used in the application. Since the Persistent Volume is located inside the Minishift VM, you can do it using the following command:
+
+```
+$ scp -i $HOME/.minishift/machines/minishift/id_rsa hazelcast.xml docker@$(minishift ip):/mnt/sda1/var/lib/minishift/openshift.local.pv/pv0001/
+```
+
+Short explanation of the command above:
+* `$HOME/.minishift/machines/minishift/id_rsa` - ssh key to Minishift VM is stored in the Minishift's home directory
+* `hazelcast.xml` - custom configuration of Hazelcast
+* `minishift ip` - command to return the IP address of the Minishift VM
+* `/mnt/sda1/var/lib/minishift/openshift.local.pv/pv0001/` - location of the Persistent Volume `pv0001` in Minishift VM
+
+The other possibility to put a configuration inside the Minishift VM is to share a directory with the host system using [Minishift hostfolder](https://docs.openshift.org/latest/minishift/using/host-folders.html).
+
+After starting the application again, the containers use the custom Hazelcast configuration.
+
+# Development Tips
+
+## Useful commands
+
+The complete guide to the `oc` CLI tool can be found [here](https://docs.openshift.org/latest/cli_reference/index.html). Below you can see the most interesting use cases in the context of Hazelcast.
+
+**Scaling application**
+
+To scale the Hazelcast application, you can change the number of replicas in the Replication Controller. For example, to scale up to 5 replicas, use the following comamnd:
+
+```
+$ oc scale rc/hz-rc --replicas=5
+```
+
+**Exposing application**
+
+By default (as mentioned in [Security Implications](#security-implications)) the Hazelcast cluster is accessible only from the OpenShift environment. You can, however, make it accessible from outside.
+
+```
+$ oc expose svc/hzservice
+route "hzservice" exposed
+```
+
+Then, you should be able to access Hazelcast via the exposed route (you can check what the route is by `oc status` or `oc get routes/hzservice`). For example, to check the health of Hazelcast:
+
+```
+$ curl hzservice-hazelcast.192.168.1.113.nip.io/hazelcast/health
+Hazelcast::NodeState=ACTIVE
+Hazelcast::ClusterState=ACTIVE
+Hazelcast::ClusterSafe=TRUE
+Hazelcast::MigrationQueueSize=0
+Hazelcast::ClusterSize=1
+```
+
+## Local Docker images
+
+During the development process, a very common use case is to build locally own Docker images and run them on Minishift. E.g. you may want to build on top of the Hazelcast OpenShift image and check if it works, or you may want to create a seprate application and check how it interacts with Hazelcast when deployed together on OpenShift.
```

Comment:
...on Minishift. For example, you may want to build..........to create a separate application....

Reply:
Done